src/main/java/com/mongodb/jdbc/MongoStatement.java

```diff
@@ -19,7 +19,6 @@
     protected int fetchSize = 0;
     protected int maxQuerySec = 0;
     protected String currentDBName;
-    protected final BsonInt32 formatVersion = new BsonInt32(2);
```

Comment:
Added change from the SQL-555 PR for the format version as it's needed to pass the evergreen tests.